src/init.cpp

```diff
@@ -465,8 +465,10 @@ std::string HelpMessage(HelpMessageMode mode)
     AppendParamsHelpMessages(strUsage, showDebug);
 
     strUsage += HelpMessageGroup(_("Node relay options:"));
-    if (showDebug)
+    if (showDebug) {
         strUsage += HelpMessageOpt("-acceptnonstdtxn", strprintf("Relay and mine \"non-standard\" transactions (%sdefault: %u)", "testnet/regtest only; ", !Params(CBaseChainParams::TESTNET).RequireStandard()));
+        strUsage += HelpMessageOpt("-incrementalrelayfee=<amt>", strprintf(_("Fee rate (in %s/kB) used to define cost of free relay, used for mempool limiting and BIP 125 replacement. (default: %s)"), CURRENCY_UNIT, FormatMoney(DEFAULT_INCREMENTAL_RELAY_FEE)));
```

Comment:
Is this not supposed to be translated?  Also, "cost of free relay" is a very confusing phrase.  Perhaps "cost of relay"?